graphql_compiler/tests/test_cost_estimation.py

```diff
@@ -798,3 +799,45 @@ def empty_lookup_counts(classname):
         )
         expected_selectivity = Selectivity(kind=ABSOLUTE_SELECTIVITY, value=1.0)
         self.assertEqual(expected_selectivity, selectivity)
+
+    @pytest.mark.usefixtures('graph_client')
+    def test_get_in_collection_filter_selectivity(self):
+        schema_graph = generate_schema_graph(self.graph_client)
+        classname = 'Animal'
+
+        def empty_lookup_counts(classname):
+            """Dummy function to pass into get_filter_selectivity."""
+            return 100
+
+        nonunique_filter = FilterInfo(fields=('birthday',), op_name='in_collection',
+                                      args=('$birthday_collection',))
+        # These params aren't relevant to _get_filter_selectivity's result, but may expose bugs
+        nonunique_params = {
+            'birthday_collection': [
+                date(2017, 3, 22),
+                date(1999, 12, 31),
+            ]
+        }
+        # If we use an in_collection-filter on a property that is not uniquely indexed, do nothing.
+        selectivity = _get_filter_selectivity(
+            schema_graph, empty_lookup_counts, nonunique_filter, nonunique_params, classname
+        )
+        expected_selectivity = Selectivity(kind=FRACTIONAL_SELECTIVITY, value=1.0)
+        self.assertEqual(expected_selectivity, selectivity)
+
+        # If we use an in_collection-filter on a property that is uniquely indexed, expect as many
+        # results as there are elements in the collection.
```

Comment:
nit: expect an absolute selectivity of the size of the collection.

Reply:
That's what reading the next few lines keyword-for-keyword already do. I feel like expressing this more informally communicates it better.